qemu/tests/cfg/block_performance_test.cfg

```diff
@@ -76,30 +76,33 @@
     pre_command = "if ! which fio;then yum install fio -y;fi &&"
     pre_command += "  mkdir -p ${fio_dir};"
 
-    fio_cmd = " --runtime=20 --ramp_time=3 --size=4G --time_based=1 --direct=1 "
+    fio_cmd = " --runtime=20 --ramp_time=5 --size=8G --time_based=1 --direct=1 "
     fio_cmd += " --numjobs=8 --group_reporting --filename=%s "
     fio_cmd += " --randrepeat=1 --output-format=json "
 
     host_init_operation = lsblk
     host_deinit_operation = host_func_demo
+    # Merge and separate(default)
+    fio_run_mode = separate
     Linux:
         fio_cmd += " --ioengine=libaio "
         guest_init_operation = lsblk -n
         guest_deinit_operation = guest_func_demo
+#        fio_iteration_cmd = "echo 3 > /proc/sys/vm/drop_caches"
```

Comment:
Just a suggestion, keep the same indentation as the previous line. like line 85.
#  fio_iteration_cmd = "echo 3 > /proc/sys/vm/drop_caches"

Reply:
updated